packages/components/ovh-shell/src/plugin/tracking/index.ts

```diff
@@ -9,7 +10,12 @@ import {
 import ShellClient from '../../client/shell-client';
 
 export interface TrackingAPI {
+  init(): PromiseLike<void>;
   isTagAvailable(): PromiseLike<boolean>;
+  clearTrackQueue(): PromiseLike<void>;
+  processTrackQueue(): PromiseLike<void>;
+  initTag(): PromiseLike<void>;
```

Comment:
i'm not sure the tracking api should know the concept of "tags" which are atinternet related, the tracking api should be as abstract as possible with only methods for tracking clicks, actions etc

Reply:
I agree with u, and at first, i only included track methods in the API. But unfortunately, by replacing the atInternet module with the tracking plugin, it turns out some of these methods were already used in other modules, which means we need to expose them so we don't break legacy code